actionview/lib/action_view/engine.rb

```diff
@@ -0,0 +1,64 @@
+require "action_view"
+require "rails"
+
+module ActionView
+  # = Action View Engine
+  class Railtie < Rails::Engine # :nodoc:
```

Comment:
How about leaving this in `lib/action_view/railtie.rb`? The fact that its superclass is now `Rails::Engine` doesn't mean it needs to move.